lib/js/components/SelectList/SelectList.consts.ts

```diff
@@ -1,3 +1,9 @@
 export const SELECT_LIST_SIZES = {
```

Comment:
Is it a const related to `SelectList` or `SelectListItem`?

Reply:
I know, but we're using it in both `SelectListItem` and `SelectListItemToggle`. I see 3 options:
1. Leave it as is.
2. Rename it to `SELECT_LIST_ITEM_SIZES` and use it it like that in `SelectListItemToggle` – IMO still confusing.
3. Split it to two – `SELECT_LIST_ITEM_SIZES` and `SELECT_LIST_ITEM_TOGGLE_SIZES` – IMO a bit overcomplicated.

Let me know what option you think is the best, I can migrate.

Reply:
I'm ok with 2., it's little bit confusing, but less than currently
Agree with 3. - overcomplicated